src/ducks/balance/components/AccountRow.jsx

```diff
@@ -134,7 +164,8 @@ class AccountRow extends React.PureComponent {
           [styles['AccountRow--hasWarning']]: hasWarning,
           [styles['AccountRow--hasAlert']]: hasAlert,
           [styles['AccountRow--disabled']]:
-            (!checked || disabled) && account.loading !== true
+            (!checked || disabled) && account.loading !== true,
+          [styles['AccountRow--withOwners']]: shouldShowOwners
```

Comment:
Nit: I am always troubled when I see this kind of code (class of the parent depending on the children), why is the content of the row not expanding automatically the row ?